test/e2e/framework/util.go

```diff
@@ -4080,3 +4080,19 @@ func UpdatePodWithRetries(client *client.Client, ns, name string, update func(*a
 	}
 	return nil, fmt.Errorf("Too many retries updating Pod %q", name)
 }
+
+func GetPodsInNamespace(c *client.Client, ns string, ignoreLabels map[string]string) ([]*api.Pod, error) {
```

Comment:
I'd ask for a godoc comment for the public method but it looks like a number of public methods in this file don't bother, so I leave it to you